angular.json

```diff
@@ -843,8 +843,5 @@
       }
     }
   },
-  "defaultProject": "demoshell",
-  "cli": {
-    "analytics": false
-  }
-}
\ No newline at end of file
+  "defaultProject": "demoshell"
```

Comment:
why are we removing it?

Reply:
@pmartinezga added it accidentally 5 days ago here: https://github.com/Alfresco/alfresco-ng2-components/pull/7652/files#diff-2d2675bb4687601a5c7ccf707455132f90f3516a33716185687e5c41df59ac7dR846-R850
We should remove that.